libs/pages/logs/environment/src/lib/feature/pod-logs-feature/pod-logs-feature.tsx

```diff
@@ -109,7 +109,7 @@ export function PodLogsFeature({ clusterId }: PodLogsFeatureProps) {
       Boolean(projectId) &&
       Boolean(environmentId) &&
       Boolean(serviceId) &&
-      !disabledLogs &&
+      disabledLogs &&
```

Comment:
should be no ?
```suggestion
      !disabledLogs &&
```

Reply:
I rename the variable to avoid confusing : https://github.com/Qovery/console/pull/999/commits/b3b32e6c80531f013917c1f25a9fc142118130dc